csharp/ql/src/utils/modeleditor/ModelEditor.qll

```diff
@@ -34,7 +34,17 @@ class Endpoint extends Callable {
    * Gets the unbound type name of this endpoint.
    */
   bindingset[this]
-  string getTypeName() { result = nestedName(this.getDeclaringType().getUnboundDeclaration()) }
+  string getTypeName() {
+    result = qualifiedTypeName(this.getNamespace(), this.getDeclaringType().getUnboundDeclaration())
+  }
+
+  /**
+   * Gets the qualified method name of this endpoint.
+   */
+  bindingset[this]
+  string getMethodName() {
```

Comment:
Maybe consider to use `getEndpointName` or `getCallableName`. The endpoint logic also covers property accessors, which are not classified as methods.

Reply:
Thanks, that makes sense. I've renamed it to `getEndpointName`.